oauth2_http/java/com/google/auth/mtls/DefaultMtlsProviderFactory.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2025, Google Inc. All rights reserved.
```

Comment:
nit: [go/copyright](https://goto.google.com/copyright), same below
```suggestion
 * Copyright 2025 Google LLC
```

Reply:
Fixed! (Used the format used by gax lib)